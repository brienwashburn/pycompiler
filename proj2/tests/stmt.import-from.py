from . import *
from . import a
from . import a, b, c
from . import a, b, c
from . import a as foo, b as boo, c as coo
from . import (a as foo, b as boo, c as coo,)
from . import (a , b , c ,)
from ..... import a
from .a.b.c import rax
from ....a.b.c import qq
from a.b.c import rax as b, foo as bar
from a import b as c, foo as bar
