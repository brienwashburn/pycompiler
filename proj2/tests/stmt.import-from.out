(Module
 (ImportFrom (module #f) (names (* #f)) (level 1))
 (ImportFrom (module #f) (names (a #f)) (level 1))
 (ImportFrom (module #f) (names (a #f) (b #f) (c #f)) (level 1))
 (ImportFrom (module #f) (names (a #f) (b #f) (c #f)) (level 1))
 (ImportFrom (module #f) (names (a foo) (b boo) (c coo)) (level 1))
 (ImportFrom (module #f) (names (a foo) (b boo) (c coo)) (level 1))
 (ImportFrom (module #f) (names (a #f) (b #f) (c #f)) (level 1))
 (ImportFrom (module #f) (names (a #f)) (level 5))
 (ImportFrom (module a.b.c) (names (rax #f)) (level 1))
 (ImportFrom (module a.b.c) (names (qq #f)) (level 4))
 (ImportFrom (module a.b.c) (names (rax b) (foo bar)) (level 0))
 (ImportFrom (module a) (names (b c) (foo bar)) (level 0)))
